<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="grView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>490</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addNode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>AddNode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delNode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>DelNode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEdge">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>510</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>AddEdge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchWidth">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>searchInWidth</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchInDepth">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>510</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>searchInDepth</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dijkstra">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>510</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Dijkstra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upd">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>510</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>â™€</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="tsp">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>510</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TSP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
